elastic/tests/test_elastic.py

```diff
@@ -202,6 +203,21 @@ def test_index_metrics(dd_environment, aggregator, instance, cluster_tags):
         aggregator.assert_metric(m_name, tags=cluster_tags + ['index_name:testindex'])
 
 
+@pytest.mark.integration
+def test_cat_allocation_metrics(dd_environment, aggregator, instance, cluster_tags):
+    instance['cat_allocation_stats'] = True
+    elastic_check = ESCheck('elastic', {}, instances=[instance])
+    cat_allocation_metrics = {}
+    cat_allocation_metrics.update(CAT_ALLOCATION_METRICS)
```

Comment:
same here, let's just use the constant directly instead of assigning to local var.